setup.cfg

```diff
@@ -0,0 +1,43 @@
+[metadata]
```

Comment:
This file is useless since we pass all of it into the `setup()` function in the *setup.py* file. What I would suggest doing is getting rid of all of the `setup()` parameters and let the setuptools query it from here.

Reply:
I removed setup.cfg. For the setup.py do you suggest reducing its content to something like that:

`#!/usr/bin/env python
import setuptools
setuptools.setup()`

Reply:
It is common practice to try and stick as much information as possible into the declarative `setup.cfg` file and keep the bare minimum in the `setup.py`. I created a demo branch [here](https://github.com/xlab-si/python-object-store-library/tree/setup) that contains a fully functioning package setup. I also rebased and squashed the fixup commits while I was at it.

Reply:
@tadeboro thanks a lot for the demo branch with example.